tests/h/util/document_claims_test.py

```diff
@@ -794,5 +794,4 @@ def one(list_):
 def strip_prefix(prefix, s):
     if s.startswith(prefix):
         return s[len(prefix) :]
-    else:
-        return s
+    return s
```

Comment:
Looks like should be hitting the invalid variable name here? Is that disabled for the tests?

Reply:
We probably are, but as I go thru this by "rule" I sometimes have to visit the same line more than once.